src/com/seafile/seadroid2/ui/activity/BrowserActivity.java

```diff
@@ -889,8 +889,17 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             break;
         case PICK_FILE_REQUEST:
             if (resultCode == RESULT_OK) {
-                Log.d(DEBUG_TAG, "Got uri: " + data.getData());
-                ConcurrentAsyncTask.execute(new SAFLoadRemoteFileTask(), data.getData());
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+                    List<Uri> uriList = UtilsJellyBean.extractUriListFromIntent(data);
+                    for (Uri uri: uriList) {
+                        Log.d(DEBUG_TAG, "Got uri: " + uri);
+                        ConcurrentAsyncTask.execute(new SAFLoadRemoteFileTask(), uri);
```

Comment:
I prefer to pass a `List<Uri>` the `SAFLoadRemoteFileTask` constructor instead of invoking several `SAFLoadRemoteFileTask`each time.